### DISCUSSION TITLE
Allure TestOps, Test Cases view: Please add a button to collapse the test tree

### DISCUSSION THREAD

**Discussion by naiad09** [allure-framework/allure2#1627] [TestOps]
Hi Allure Team!

The Test Cases view is a two-side panel, so I can drag-n-drop the splitter to change the test tree / test area width.

Additionaly to that, I think it would be convenient to have a collapse/expand button, so users can collapse/expand the test cases tree.

**Comment by cheshi-mantu** [comment]
> Hi Allure Team!
> 
> The Test Cases view is a two-side panel, so I can drag-n-drop the splitter to change the test tree / test area width.
> 
> Additionaly to that, I think it would be convenient to have a collapse/expand button, so users can collapse/expand the test cases tree.

Hi there.
Thanks for writing your suggestions. We really appreciate it!
Although, the feature requests, bugs and support requests must go via https://help.qameta.io only. This is the only way to suggest an improvement.
